### PR TITLE
Panel figure enhancements

### PR DESCRIPTION
**Pull request by philschanely**
## Description

This PR separates the Panel Figure component from the Panel page into its own page to improve documentation and adds the following new features (non-destructive):

- Images can be set up with a specific aspect ratio.
- A background color can be provided for behind the image.
- Images can be padded to reveal a small "frame" of the background color.

## Screenshots

Screen Shot 2021-07-13 at 12 04 35 PM

Screen Shot 2021-07-13 at 12 04 53 PM

Screen Shot 2021-07-13 at 12 05 05 PM

## Testing in `sage-lib`

See new Panel Figure page (Rails only ATM).

## Testing in `kajabi-products`

(LOW) New customization options added to Panel Figure. No impact on current uses.

### CODE REVIEW

File: docs/app/views/examples/components/panel_figure/_preview.html.erb [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  <%= sage_component SagePanel, {} do %>
+    <p>In larger or tall panels a figure may sit amidst other content.</p>
+    <%= sage_component SagePanelFigure, { bleed: "sides" } do %>
+      <%= image_tag("card-placeholder-lg.png", alt: "") %>
+    <% end %>
+    <p>In such cases a side bleed option can be used.</p>
+  <% end %>
+
+  <%= sage_component SagePanel, {} do %>
+    <p>Less frequently, a figure at the very bototm of a panel may benefit from bleeding to those lower edges.</p>
```
Comment (monicawheeler) [inline docs/app/views/examples/components/panel_figure/_preview.html.erb:41]: Spelling of "bottom" :)

File: docs/app/views/examples/components/panel_figure/_preview.html.erb [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+  <%= sage_component SagePanel, {} do %>
+    <%= sage_component SagePanelFigure, { bleed: "top" } do %>
+      <%= image_tag("card-placeholder-lg.png", alt: "") %>
+    <% end %>
+    <p>A common pattern for a figure at the top of a panel is for it to be set to bleed to the edges of the panel.</p>
+  <% end %>
+
+  <%= sage_component SagePanel, {} do %>
+    <p>In larger or tall panels a figure may sit amidst other content.</p>
```
Comment (monicawheeler) [inline docs/app/views/examples/components/panel_figure/_preview.html.erb:33]: "In larger or **taller** panels, a figure may sit **among** other content"

File: packages/sage-assets/lib/stylesheets/components/_panel.scss [hunk 3]
```diff
@@ -42,46 +48,7 @@
...
-  width: calc(100% + #{2 * sage-spacing()});
-  margin: (-1 * sage-spacing()) (-1 * sage-spacing()) 0;
-  border-bottom-left-radius: 0;
-  border-bottom-right-radius: 0;
-}
-
-.sage-panel__figure--wistia {
-  display: block;
-}
+// Panel Figure moved to its own file
```
Comment (monicawheeler) [inline packages/sage-assets/lib/stylesheets/components/_panel.scss:51]: 🙌🏻

**Review by monicawheeler — APPROVED** [review]
Small updates to documentation but other than that 👍

File: docs/app/views/examples/components/panel_figure/_preview.html.erb [hunk 4]
```diff
@@ -0,0 +1,91 @@
+<%
+sample_copy = md("
+Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+Morbi rhoncus mauris ut turpis gravida bibendum.
+Donec feugiat accumsan velit et semper.
+Ut elementum id leo ac fringilla.
+")
+%>
```
Comment (pixelflips) [inline docs/app/views/examples/components/panel_figure/_preview.html.erb:8]: Can this be removed? Not seeing `sample_copy` used in the file.

**Review by pixelflips — COMMENTED** [review]
Left one minor non-blocking comment but overall LGTM! Nice work! 👍🏼

File: docs/app/views/examples/components/panel_figure/_preview.html.erb (hunk 4 above)
Comment (philschanely) [reply docs/app/views/examples/components/panel_figure/_preview.html.erb:8]: Will clean up in a followup

**Pull request merged**